tools/windows_buildenv.bat

```diff
@@ -62,13 +65,17 @@ EXIT /B 0
             REM TODO: verify download using sha256sum?
         )
         ECHO ### Unpacking !BUILDENV_PATH!.zip ###
-        CALL :UNZIP "!BUILDENV_PATH!.zip" "!BUILDENV_BASEPATH!"
+        CALL :DETECT_7ZIP
+        IF "%RETVAL%" EQ "" (
```

Comment:
```suggestion
        IF "%RETVAL%" == "" (
```
@JoergAtGithub Would this work?

Reply:
This seems to work 
![grafik](https://user-images.githubusercontent.com/64457745/121603674-6c1a0500-ca49-11eb-81a5-ba3611b4f0bb.png)

Reply:
```suggestion
        IF NOT DEFINED RETVAL (
```

Reply:
The batch file just suceeded with the Powershell version (with optional installation of Powershell version 5.1 under Win7).

Reply:
So powershell is not installed on Win7 by default? But I don't think that's a problem, Win7 reached EOL 1.5 years ago.

Reply:
It comes with an older version of Powershell. I tested only with the newer 5.1 powershell.